docs/source/federation-spec.md

```diff
@@ -309,3 +309,48 @@ extend type User @key(fields: "email") {
 ```
 
 This type extension in the Reviews service extends the `User` type from the Users service. It extends it for the purpose of adding a new field called `reviews`, which returns a list of `Review`s.
+
+### `@shareable`
+
+```graphql
+directive @shareable on FIELD_DEFINITION | OBJECT
+```
+
+The `@shareable` directive denotes that a field may be resolved by multiple subgraphs. Queries to a shareable field may potentially resolve to any subgraph which resolves the field. In order to successfully compose, a field must have the same shareability mode (either shareable or non-shareable) across all subgraphs.
+
+Any field specified by a @key is automatically shareable. Adding the `@shareable` directive to an object is equivalent to marking each field on the object `@shareable`.
+
+```graphql
+type Product @key(fields: "upc") {
+  upc: UPC!                         # shareable because upc is a key field
+  name: String                      # non-shareable
+  description: String @shareable    # shareable
+}
+
+type User @key(fields: "email") @shareable {
+  email: String                    # shareable because User is marked shareable
+  name: String                     # shareable because User is marked shareable
+}
+```
+
+### `@override`
+
+```graphql
+directive @override(from: String!) on FIELD_DEFINITION
+```
+
+The `@override` directive is used to annotate that the field in the current subgraph is taking responsibility for providing the value of the field away from the subgraph specified in the `from` argument. The following example will result in all query plans made to resolve `User.name` to be directed to `SubgraphB`.
```

Comment:
I tweaked the first sentence and would suggest separating these two sentences, like so:
```suggestion
The `@override` directive is used to indicate that the current subgraph is taking responsibility for resolving the marked field _away_ from the subgraph specified in the `from` argument. 

The following example will result in all query plans made to resolve `User.name` to be directed to SubgraphB.
```